src/ai/ai.cpp

```diff
@@ -1220,19 +1239,19 @@ crisis_str estimate_crisis_str(sys::state& state) {
 	}
 
 	if(secondary_attacker && secondary_attacker != state.primary_crisis_attacker) {
-		atotal += float(state.world.nation_get_military_score(secondary_attacker));
+		atotal += estimate_defensive_strength(state, secondary_attacker);
 	}
 	if(secondary_defender && secondary_defender != state.primary_crisis_defender) {
-		dtotal += float(state.world.nation_get_military_score(secondary_defender));
+		dtotal += estimate_defensive_strength(state, secondary_defender);
 	}
 	for(auto& i : state.crisis_participants) {
 		if(!i.id)
 			break;
 		if(!i.merely_interested) {
 			if(i.supports_attacker) {
-				atotal += float(state.world.nation_get_military_score(i.id));
+				atotal += estimate_defensive_strength(state, i.id);
```

Comment:
This is wrong: it will add in strength of allies that won't be called in to a crisis war